src/psyclone/domain/nemo/transformations/nemo_loop_fuse.py

```diff
@@ -109,16 +109,17 @@ def validate(self, node1, node2, options=None):
             if var_info1.is_read_only() and var_info2.is_read_only():
                 continue
 
+            # TODO #1213 - try to find symbol in case of 'wildcard' imports.
             try:
                 # Find the symbol for this variable. We only need to check
```

Comment:
I didn't notice this before but what does "We only need to check one symbol table" mean?

Reply:
That you don't need to look manually at outer scopes - but agrees, that was bad wording, I have changed that.